uisdk/src/main/java/tech/dojo/pay/uisdk/presentation/ui/virtualterminalcheckout/BillingAddressSection.kt

```diff
@@ -115,31 +111,26 @@ private fun Address1Field(
     coroutineScope: CoroutineScope,
     scrollToPosition: Float,
     scrollState: ScrollState,
-    keyboardController: SoftwareKeyboardController?
+    keyboardController: SoftwareKeyboardController?,
 ) {
-    var isTextNotFocused by remember { mutableStateOf(false) }
+    val hasBeenFocused by remember { mutableStateOf(false) }
     val scrollOffset = with(LocalDensity.current) {
         billingAddressViewState.itemPoissonOffset.dp.toPx() + NORMAL_FILED_SIZE_DP.dp.toPx()
```

Comment:
Can you rename the `itemPoissonOffset`
```suggestion
        billingAddressViewState.itemPositionOffset.dp.toPx() + NORMAL_FILED_SIZE_DP.dp.toPx()
```